cards.js

```diff
@@ -16,8 +17,12 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
 
-// Put some code in here!
 
-}
+function buildDeck(n){
+  for(var i = 0; i < n; i ++)
+    for(var j = 0; j < suits.length; j ++)
+      for(var k = 0; k < values.length; k ++)
+          new Card(values[k], suits[j]);
```

Comment:
I'm not sure what `new Card` does in your code, i think if you defined a `Card` function in which you pass in a suit and a value like you did on line 26 then this could work.